android/src/main/java/com/capacitorjs/osinappbrowser/InAppBrowserPlugin.kt

```diff
@@ -88,43 +57,24 @@ class InAppBrowserPlugin : Plugin() {
                     },
                     onBrowserFinished = {
                         notifyListeners(OSIABEventType.BROWSER_FINISHED.value, null)
-                    }
+                    },
+                    onBrowserPageNavigationCompleted = {}
```

Comment:
Per the recent updates in the native library, this should be removed:

```suggestion
                    }
```